### DISCUSSION TITLE
Unexpected token

### DISCUSSION THREAD

**Discussion by joshyu** [lit/lit#4652] [Q&A]
I use babelify+browserify+aliasify in my project, and also import lit. but I encountered some errors. I will provide some context here.

the entry.js file:
`import { html } from 'lit-element';`

the gulp build script:
```javascript
gulp.task('build-js', () => {
  return browserify({debug: true})
    .transform('babelify', {
      presets: ['@babel/preset-env'],
      plugins: ['@babel/plugin-transform-class-properties'],
      global: true,
    })
    .transform('aliasify', { .....})
```

the error I encountered:
```
error Babel: Unexpected token (94:46) (while aliasify was processing /app/node_modules/lit-element/lit-element.js) while parsing file: /app/node_modules/lit-element/lit-element.js
```

As `lit-element.js` is a compressed file, so I can't find out what **unexpected token** here mentioned is.

**npm versions:**
babelify: 10.0.0
aliasify: 2.1.0
@babel/core: 7.9.0

Can anyone help me how to resolve this error?

**Comment by justinfagnani** [comment]
I don't think we can do much without knowing what token is unexpected. Even in a minified file you should be able to see the bad keyword or character.

Is it possible that aliasify doesn't support optional chaining or nullish coalescing assignment?

**Reply by joshyu** [reply]
Hi @justinfagnani ,

Thank you for your prompt reply. My team mate found that we can install `@babel/parser`, then the detailed error message will be displayed.  We found that it is due to the lack of babel plugin `@babel/plugin-proposal-logical-assignment-operators`, after enable it, everything passed.
